tf_files/aws/csoc_common_logging/variables.tf

```diff
@@ -31,3 +31,15 @@ variable "threshold" {
 variable "slack_webhook" {
  default = ""
 }
+
+variable "log_dna_function" {
+  default = "arn:aws:lambda:us-east-1:433568766270:function:logdna_cloudwatch"
+}
```

Comment:
how does this get setup?
we should have a separate script to publish the logdna lambda, and track that code?

Reply:
The logDNA function was something created by the security team following logDNA documentation. It is a one time thing only and there is no need to code it. Very unlikely to be modified in the near future unless logDNA releases a new version or something.